src/core/Akka.TestKit/TestProbe.cs

```diff
@@ -184,7 +184,7 @@ public void SendSystemMessage(ISystemMessage message, IActorRef sender)
             CancellationToken cancellationToken = default)
             where T : ActorBase
             => ChildActorOfAsync(Props.Create<T>(), name, supervisorStrategy, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.